### PR TITLE
V0.14.0 dev

### PR DESCRIPTION
**Pull request by jwildfire**
Adds analysisFlag setting and provides some better reporting on missing data. 

Project [here](https://github.com/ASA-DIA-InteractiveSafetyGraphics/safety-eDISH/projects/12)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,11 @@
...
   "devDependencies": {
     "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.6.0",
+    "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "prettier": "^1.7.4",
+    "prettier": "^1.14.3",
     "rollup": "^0.50.0",
-    "rollup-plugin-babel": "^3.0.2",
-    "rollup-watch": "^4.3.1"
```
Comment (samussiah) [inline package.json]: but why?

Comment (jwildfire) [reply package.json]: rollup-watch is deprecated and uses the insecure dependency described in #169

File: src/callbacks/onInit/cleanData/makeAnalysisFlag.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export default function makeAnalysisFlag() {
+    var chart = this;
+    var config = this.config;
+    console.log(chart.imputed_data);
+    this.imputed_data = this.imputed_data.map(function(d) {
+        const hasAnalysisSetting =
+            (config.analysisFlag.value_col != null) & (config.analysisFlag.values.length > 0);
```
Comment (samussiah) [inline src/callbacks/onInit/cleanData/makeAnalysisFlag.js]: single ampersand

File: src/callbacks/onResize/addPointClick/measureTable/lineChart/onResize/drawNormalRange.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export function drawNormalRange() {
             return { visitn: m.visitn, value: m.lln };
         })
         .reverse();
-    var normal_data = d3.merge([upper, lower]);
+    var normal_data = d3.merge([upper, lower]).filter(f => f.value);
```
Comment (samussiah) [inline src/callbacks/onResize/addPointClick/measureTable/lineChart/onResize/drawNormalRange.js]: what if the lower limit is 0?

Comment (jwildfire) [reply src/callbacks/onResize/addPointClick/measureTable/lineChart/onResize/drawNormalRange.js]: well ... that would be a problem!

Comment (jwildfire) [reply src/callbacks/onResize/addPointClick/measureTable/lineChart/onResize/drawNormalRange.js]: This should fix it. 

```suggestion
    var normal_data = d3.merge([upper, lower]).filter(f => f.value || f.value==0);
```

File: src/callbacks/onInit/cleanData/makeAnalysisFlag.js (hunk 2 above)
Comment (jwildfire) [reply src/callbacks/onInit/cleanData/makeAnalysisFlag.js]:
```suggestion
            (config.analysisFlag.value_col != null) && (config.analysisFlag.values.length > 0);
```

**Pull request merged**